pkg/test/verify/cluster.go

```diff
@@ -75,13 +70,12 @@ func AddToolchainClusterAsHost(t *testing.T, functionToVerify FunctionToVerify)
 	defer gock.Off()
 	status := test.NewClusterStatus(toolchainv1alpha1.ToolchainClusterReady, corev1.ConditionFalse)
 	memberLabels := []map[string]string{
-		Labels(cluster.Host, "", test.NameMember),
-		Labels(cluster.Host, "host-ns", test.NameMember)}
+		Labels(test.MemberOperatorNs, test.NameMember),
```

Comment:
shouldn't this case pass an empty namespace
```suggestion
		Labels("", test.NameMember),
```

Reply:
yes.. updated it